### ISSUE TITLE
Image refresh for rhel-7-7

### ISSUE THREAD

**Issue by cockpituous** [cockpit-project/bots#276]
Image refresh for rhel-7-7
 * [ ] FAIL: image-refresh rhel-7-7

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-10.
Log: https://209.132.184.41:8493/logs/image-refresh-276-20191125-131515/

**Comment by cockpituous** [comment]
Task failed: https://209.132.184.41:8493/logs/image-refresh-276-20191125-131515/

**Comment by martinpitt** [comment]
@Gundersanne: Looks like this new package name needs to be blacklisted on rhel-7-7?
```
No package glibc-all-langpacks available.
Error: Not tolerating missing names on install, stopping.
```
Curiously 7.8 seems to work, see #255

**Comment by martinpitt** [comment]
@Gundersanne : ah no, the previous 7-8 refresh just ran before glibc-all-langpacks got added. #255 failed on the same issue now.

**Comment by martinpitt** [comment]
Fixed in PR #280
